ceph-pull-requests/config/definitions/ceph-pull-requests.yml

```diff
@@ -62,3 +62,22 @@
                   - ABORTED
               build-steps:
                 - shell: "sudo reboot"
+      - cobertura:
+          report-file: "src/pybind/mgr/dashboard/frontend/coverage/cobertura-coverage.xml"
+          only-stable: "true"
+          fail-no-reports: "true"
```

Comment:
do you really want to fail the build if no reports are produced?

Reply:
If build passes, report will be there for sure, so if report is not there, means build fails. So I thought of putting this condition. It should be fine if I remove it as well.

Reply:
Just making sure that is really the intention here. I'm OK if that is the objective